configure.ac

```diff
@@ -724,7 +735,8 @@ RKT_IF_HAS_FLAVOR([${RKT_STAGE1_FLAVORS}], [coreos,kvm],
         coreos/kvm flavor specific build parameters
 
         local CoreOS PXE image path:            '${RKT_LOCAL_COREOS_PXE_IMAGE_PATH}'
-        local CoreOS PXE image systemd version: '${RKT_LOCAL_COREOS_PXE_IMAGE_SYSTEMD_VER}'])])
+        local CoreOS PXE image systemd version: '${RKT_LOCAL_COREOS_PXE_IMAGE_SYSTEMD_VER}'
+        stage1 CoreOS board:                    '${RKT_STAGE1_COREOS_BOARD}'])])
```

Comment:
Specifically inherited from ChromeOS where each image type literally is targeting a specific piece of physical hardware. Our image types are more general purpose so it makes less sense but the term is used so extensively it wouldn't really be worth changing.